tests/scripts/docker.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -eu
```

Comment:
I think this script could use a better name, like `prepare_docker.sh` or `docker_env.sh`.

Reply:
Done (docker_env.sh)